packages/widget/README.md

```diff
@@ -0,0 +1,55 @@
+# Sygmaprotocol Lit Widget
+
+### Dependencies
+ 
+To integrate the Widget to any lit project that you have, you will need to add the following dependencies:
+
+```bash
+yarn add @buildwithsygma/sygmaprotocol-widget @buildwithsygma/sygma-sdk-core
+```
+
+### Code integration
+
+To add the Wdiget to your existing codebase just import the dependency and add the custom tag into your render method.
+
+```ts
+import { LitElement, html } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import '@buildwithsygma/sygmaprotocol-widget'
+
+@customElement('my-element')
+export class MyElement extends LitElement {
```

Comment:
I think mentioning lit is confusing. You don't need lit to use web component: https://www.webcomponents.org/introduction#how-do-i-use-a-web-component-

Reply:
Since the widget is a Lit codebase I thought it make sense, but we can remove this